src/settings-ui/Settings.UI/Flyout/ShellPage.xaml

```diff
@@ -8,7 +8,6 @@
     Loaded="Page_Loaded"
     mc:Ignorable="d">
 
-    <Grid>
-        <Frame x:Name="ContentFrame" />
-    </Grid>
+    <Frame x:Name="ContentFrame" 
+           Background="{ThemeResource SystemControlAcrylicWindowBrush}" />
```

Comment:
I am confused why we need this.. the FlyoutWindow itself already has the background acrylic defined. We should make any theme changes there?

Also, I think the `SystemControlAcrylicWindowBrush` is an OS brush. The design spec requires us to use: `AcrylicBackgroundFillColorBaseBrush`

![image](https://user-images.githubusercontent.com/9866362/221412155-765b55c2-7fbc-47c8-a994-75c466c672eb.png)

Reply:
Both should have just been fixed with my latest commits.